alternator/rjson.cc

```diff
@@ -51,9 +51,23 @@ namespace rjson {
 
 static allocator the_allocator;
 
+class yieldable_writer : public writer {
+public:
+    yieldable_writer(string_buffer& buf) : writer(buf) {}
+    bool Null()                 { maybe_yield(); return writer::Null(); }
```

Comment:
Nitpick: all of these functions call EndValue(), so I wonder if just changing EndValue() once, instead of all of these, would have been enough?